interbotix_ros_xscobots/interbotix_xscobot_moveit/launch/xscobot_moveit.launch.py

```diff
@@ -250,8 +251,8 @@ def launch_setup(context, *args, **kwargs):
             'robot_model': robot_model_launch_arg,
             'robot_name': robot_name_launch_arg,
             'base_link_frame': base_link_frame_launch_arg,
-            # 'show_ar_tag': show_ar_tag_launch_arg,
-            # 'show_gripper_bar': 'true',
+            'show_ar_tag': show_ar_tag_launch_arg,
+            # # 'show_gripper_bar': 'true',
```

Comment:
This line is double commented
```suggestion
            # 'show_gripper_bar': 'true',
```